<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>661</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_Main">
     <item row="0" column="0">
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>7</number>
       </property>
       <widget class="QWidget" name="page_Auth">
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>70</y>
           <width>551</width>
           <height>311</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_AuthMain">
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_AuthPassword">
            <property name="placeholderText">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton_Authorize">
            <property name="text">
             <string>Authorize</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_AuthUsername">
            <property name="placeholderText">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_Menu">
        <widget class="QWidget" name="gridLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>90</y>
           <width>521</width>
           <height>301</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_Meny"/>
        </widget>
       </widget>
       <widget class="QWidget" name="page_Opniion">
        <widget class="QWidget" name="gridLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>70</y>
           <width>501</width>
           <height>281</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_Opninion"/>
        </widget>
       </widget>
       <widget class="QWidget" name="page_MakeAppit">
        <widget class="QWidget" name="gridLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>50</y>
           <width>581</width>
           <height>351</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_MakeAppoit"/>
        </widget>
       </widget>
       <widget class="QWidget" name="page_MyAppoits">
        <widget class="QWidget" name="gridLayoutWidget_6">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>90</y>
           <width>441</width>
           <height>261</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_MyAppoits"/>
        </widget>
       </widget>
       <widget class="QWidget" name="page_AddPatient">
        <widget class="QWidget" name="gridLayoutWidget_7">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>30</y>
           <width>561</width>
           <height>361</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_AddPatient"/>
        </widget>
       </widget>
       <widget class="QWidget" name="page_AddDoctor">
        <widget class="QWidget" name="gridLayoutWidget_8">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>40</y>
           <width>461</width>
           <height>291</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_AddDoctor"/>
        </widget>
       </widget>
       <widget class="QWidget" name="page_OutpatientCards">
        <widget class="QWidget" name="gridLayoutWidget_9">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>160</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_OutpatientCards"/>
        </widget>
       </widget>
       <widget class="QWidget" name="page_Settings">
        <widget class="QWidget" name="gridLayoutWidget_10">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>60</y>
           <width>160</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_Settings"/>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_AuthUsername</tabstop>
  <tabstop>lineEdit_AuthPassword</tabstop>
  <tabstop>pushButton_Authorize</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
